docs/getting-started/getting_started_14.md

```diff
@@ -0,0 +1,110 @@
+In order to use icons with GTK, we have two options : 
+
+- Use icons bundled with GTK.
+- Bundle SVG icons within our app resources.
+
+### GTK bundled icons
+
+To use GTK bundle icons, we just need to know the name of the icon we want to use. We can use the "GTK Icon Browser" app to see the icons installed on our system.
+
+To install it, run :
+
+=== "Ubuntu/Debian"
+
+    ```bash
+    apt-get install gtk-3-examples
+    ```
+
+=== "Fedora/RHEL"
+
+    ```bash
+    dnf install gtk3-devel
+    ```
+
+
+We can then look up icons and call them by name in our code. For instance :
+
+```Java
+var icon = Image.fromGicon(Icon.newForString("avatar-default-symbolic"));
+```
+
+This is easy but there aren't a lot of icons to choose from
+
+### Custom SVG Icons
+
+#### Get an SVG icon Source
+
+A good source of GTK-themed SVG icons is the [Icons Library app](https://flathub.org/apps/org.gnome.design.IconLibrary). The app offers a lot of icons and allows copying the SVG code.
+
+Let's say we want to use the `penguin-symbolic` icon from the [Icons Library app](https://flathub.org/apps/org.gnome.design.IconLibrary). Let's save its SVG code in a file in our Java resource folder (i.e. `src/main/resources/icons`).
+
+#### GTK resources compiler
+
+First, we install the software to compile GTK application resources : 
+
+=== "Ubuntu/Debian"
+
+    ```bash
+    apt-get install libglib2.0-dev-bin
+    ```
+
+=== "Fedora/RHEL"
+
+    ```bash
+    dnf install glib2-devel-2.82.2-1.fc41.x86_64
```

Comment:
`dnf install glib2-devel` should do the trick. The additional version info isn't needed and will quickly be outdated